test/y2storage/volume_specification_test.rb

```diff
@@ -51,6 +50,17 @@
         expect(described_class.for("/")).to be_nil
       end
     end
+
+    context "when a volume specification was already calculated" do
+      before do
+        described_class.for("/")
+      end
+
+      it "retrieves the value from the internal cache" do
+        expect(Y2Storage::VolumeSpecificationBuilder).to_not receive(:new)
+        described_class.for("/")
+      end
+    end
```

Comment:
It would be nice to have a test for `#clear_cache`.

Reply:
I was taking a shortcut because it is used in the before block for the `for` method. But you are right, a proper test should be used.